package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-data-github",
-  "version": "0.0.6",
+  "version": "1.0.0",
```

Comment:
Please revert. I'll update the package version via `ember-cli-release` when the new version is ready to go out. Since we're in a beta version still, this will probably be a 0.1.0 release rather than 1.0.0, but either way I'll make that change in a separate commit.